library/blivet.py

```diff
@@ -325,6 +325,8 @@ def _resize(self):
                 raise BlivetAnsibleError("volume '%s' cannot be resized from %s to %s: %s" % (self._device.name,
                                                                                               self._device.size,
                                                                                               size, str(e)))
+        elif size and self._device.size != size and not self._device.resizable:
```

Comment:
Hi
I'm trying to cover this line, could you give me more info about on what condition the device will not be resizeable?

Thanks
Yi

Reply:
>  I cannot found the delete option for this comment.

Click the three dots in the top right corner?

Reply:
> I'm trying to cover this line, could you give me more info about on what condition the device will not be resizeable?

Are you trying to test the problems that this commit fixes, or those that it introduces? The latter is a test failure  "volume 'luks-vdb1' cannot be resized from 4 GiB to 4 GiB". Maybe wait for developers to fix it before developing tests?

Reply:
> > I cannot found the delete option for this comment.
> 
> Click the three dots in the top right corner?

I tried, but no that option. )
![no-delete](https://user-images.githubusercontent.com/7991542/86459709-cbf82e00-bd59-11ea-928c-ef5de8d65506.png)

Reply:
> > I'm trying to cover this line, could you give me more info about on what condition the device will not be resizeable?
> 
> Are you trying to test the problems that this commit fixes, or those that it introduces? The latter is a test failure "volume 'luks-vdb1' cannot be resized from 4 GiB to 4 GiB". Maybe wait for developers to fix it before developing tests?

I'm trying to test the the problems that this commit fixed.
OK, will skip this commit.

Reply:
@yizhanglinux The test failure "volume 'luks-vdb1' cannot be resized from 4 GiB to 4 GiB" is now fixed in #125 , can you please continue working on that PR?